layouts/main.tsx

```diff
@@ -112,6 +115,14 @@ const Layout = ({ children, title = "Livepeer Explorer" }) => {
     [currentRound?.currentL1Block, pollData]
   );
 
+  const totalActiveProposals = useMemo(() => {
+    const currentBlock = currentRound?.id ? parseInt(String(currentRound.id)) : 0;
+    return proposalsData?.treasuryProposals.reduce((count, p) => {
+      const voteEndBlock = Number(p.voteEnd);
+      return voteEndBlock > currentBlock ? count + 1 : count;
```

Comment:
The voting actually goes on from the start round (`voteStart`) _exclusively_ until the end round (`voteEnd`) _inclusively_. So this check should be `voteEndBlock >= currentBlock` instead.
ref: https://github.com/OpenZeppelin/openzeppelin-contracts-upgradeable/blob/v4.9.2/contracts/governance/GovernorUpgradeable.sol#L194

Reply:
Great catch! Thanks for sharing the contract, it really helps clarify things.